10.9-libcxx/gettext-tools.info

```diff
@@ -60,11 +60,13 @@ CompileScript: <<
 	#!/bin/sh -ev
 	mkdir bin
 	ln -s /usr/bin/javac bin/gcj
-	cd libtextstyle
+	pushd libtextstyle
 	env EMACS=no JAVA=/usr/bin/java PATH=../bin:$PATH ./configure --prefix=%b/localtree --enable-shared=no ac_cv_prog_AWK=/usr/bin/awk ac_cv_path_GREP=/usr/bin/grep ac_cv_path_SED=/usr/bin/sed
 	/usr/bin/make -w
-	cd ..
-	cd gettext-tools
+	pushd ../gettext-runtime
+	env PATH=../bin:%b/localtree/bin:$PATH ./configure %c
```

Comment:
Do we need to make sure in this ./configure call that no utils from %p are used (SED, AWK, GREP, etc), like we do with the other subdirectories (libtextstyle and gettext-tools) ?

Reply:
I copied this from the existing call for gettext-tools (below), which did not set any of the ac_cv_path_*, no idea if it would be required for safety there or in the new gettext-runtime build. Normally there should be no pre-existing %p in a bootstrap, but I take your point that those paths could still be left over from an existing Fink installation.